calico_node/startup/startup.go

```diff
@@ -788,12 +788,12 @@ func ensureDefaultConfig(cfg *api.CalicoAPIConfig, c *client.Client, node *api.N
 
 	// Store the Calico Version as a global felix config setting.
 	if err := c.Config().SetFelixConfig("CalicoVersion", "", VERSION); err != nil {
-		return err
+		return checkConflictError(err)
```

Comment:
If there's a conflict, I don't think we want to short-circuit here, we still want to continue with the rest of the default configuration.

Reply:
Ah, yes you're right, it shouldn't do that.